pwvlt-cli/src/main.rs

```diff
@@ -22,29 +22,35 @@ fn main() {
                 .short("g")
                 .long("get")
                 .help("Copy the password for <service> to the kill ring.")
-                .value_names(&["service", "username"])
-                .min_values(2)
-                .max_values(2),
+                .value_names(&["service"]),
         )
         .arg(
             Arg::with_name("set")
                 .short("s")
                 .long("set")
-                .help("Set password for <service> <username>.")
-                .value_names(&["service", "username"])
-                .min_values(2)
-                .max_values(2),
+                .help("Set password for <service>.")
+                .value_names(&["service"]),
+        )
+        .arg(
+            Arg::with_name("set-default")
+                .short("d")
+                .long("set-default")
+                .help("Set the default <username> for <service>.")
+                .value_names(&["service", "username"]),
         )
         .group(
             ArgGroup::with_name("cmd")
                 .required(true)
-                .args(&["set", "get"]),
+                .args(&["set", "get", "set-default"]),
         )
         .get_matches();
     let pv = PasswordVault::new(config::load_config().unwrap());
     if let Some(mut values) = matches.values_of("get") {
         let service = values.next().unwrap();
-        let username = values.next().unwrap();
+        let username = match pv.default(&service) {
+            Some(username) => username.to_string(),
+            None => prompt_string(format!("Enter username for {}", service)),
```

Comment:
`prompt_string` should probably `trim` the string it returns.

Reply:
nice one, completely missed that!